erdkotlin/src/main/java/com/elrond/erdkotlin/domain/vm/SmartContractOutput.kt

```diff
@@ -0,0 +1,33 @@
+package com.elrond.erdkotlin.domain.vm
+
+import com.google.gson.internal.LinkedTreeMap
+import java.math.BigInteger
+
+data class SmartContractOutput(
+    val returnData: List<String>?,
+    val returnCode: String,
+    val returnMessage: String?,
+    val gasRemaining: BigInteger,
+    val gasRefund: BigInteger,
+    val outputAccounts: LinkedTreeMap<String, OutputAccount>
+) {
+    data class OutputAccount(
+        val address: String,
+        val nonce: Long,
+        val balance: BigInteger?,
+        val balanceDelta: BigInteger,
+        val storageUpdates: LinkedTreeMap<String, StorageUpdate>?,
+        val callType: Long,
+
+        // documentation is not clear about this
```

Comment:
Please add a comment with a link towards `output.go` instead, for the moment:

https://github.com/ElrondNetwork/elrond-go/blob/master/core/vmcommon/output.go

Reply:
Done 👍